src/Tasks/ContentReviewEmails.php

```diff
@@ -159,4 +183,19 @@ protected function getTemplateVariables($recipient, $config, $pages)
             'ToEmail' => $recipient->Email,
         ];
     }
+
+    /**
+     * Check validity of email
+     */
+    protected function isValidEmail(?string $email): bool
+    {
+        if (!$email
+            || empty($email)
+            || !filter_var($email, FILTER_VALIDATE_EMAIL)
+        ) {
+            return false;
+        }
+
+        return true;
```

Comment:
```suggestion
        return (bool) filter_var($email, FILTER_VALIDATE_EMAIL);
```
We can let `filter_var` handle null and empty string values, which will both return false.

Reply:
Thank you, didn't think about this. DONE.